articles/deploying-crowdstrike-with-fleet.md

```diff
@@ -0,0 +1,139 @@
+
+### Installer + script
+
+After downloading the latest CrowdStrike Falcon `.msi` installer and retrieving your Customer ID from your admin console, navigate to the **Software** tab in Fleet, then select **Add software** > **Custom package**. Upload the installer from the previous step. Select **Automatic install** or **Self-service** if those options apply to your environment. 
+
+The CrowdStrike Falcon tenant needs to collect the Customer ID from the host at time of install which we can be done with an **Install Script**. Copy and paste the code snippet below in Fleet and populate the value of the `$FalconCid` variable with your Customer ID string:
+
+```
+# Set your Customer ID here
+$FalconCid = "YOUR-CUSTOMER-ID-HERE"
+
+$logFile = "${env:TEMP}/fleet-install-software.log"
+try {
+$installProcess = Start-Process msiexec.exe `
+  -ArgumentList "/quiet /norestart /lv ${logFile} /i `"${env:INSTALLER_PATH}`" CID=${FalconCid}" `
+  -PassThru -Verb RunAs -Wait
+Get-Content $logFile -Tail 500
+Exit $installProcess.ExitCode
+} catch {
+  Write-Host "Error: $_"
+  Exit 1
+}
+```
+
+There are several other optional flags that can be added. Check the documentation for a list of supported options and their functionality.
+
+## Linux
+
+As in previous steps, find the latest installer for your Linux distro and **Download**.
+
+From the **Software** tab in Fleet, select **Add software** > **Custom package**. Upload the installer. Select **Automatic install** or **Self-service** if those options apply to your environment.
+
+### Post-install script
+
+The default install script that is populated in Fleet is sufficient, but, a post-install script is needed to set the site token and start the agent services. Below is an example post-install script that will set the token, start the service and check the status. Adjust the sleep time if needed. Also, populate the value of the `$FalconCid` variable with your Customer ID string.
+
+```
+#!/bin/bash
+
+# Set your Customer ID here
+FalconCid = "YOUR-CUSTOMER-ID-HERE
+
+echo "Setting CrowdStrike Falcon Customer ID: $FalconCid"
+
+# Set the Customer ID
+sudo /opt/CrowdStrike/falconctl -s --cid="$FalconCid"
+
+# Check if the command was successful
+if [ $? -eq 0 ]; then
+    echo "Customer ID set successfully!"
+    
+    # Verify the setting
+    echo "Verifying Customer ID..."
+    sudo /opt/CrowdStrike/falconctl -g --cid
+else
+    echo "Error: Failed to set Customer ID"
+    exit 1
+fi
+```
+
+Admins can verify the installation by running a command searching for the falcon sensor with the following command:
+
+`sudo ps -e | grep falcon-sensor`
+
+## Conclusion
+
+Fleet offers admins a straight-forward approach to deploying the CrowdStrike Falcon application across your macOS, Linux and Windows hosts.
+
+Want to learn more or need additional information? Reach out directly to me or the [team at Fleet](https://fleetdm.com/contact) today!
```

Comment:
```suggestion

```

Reply:
I think we can remove this CTA since this is a technical guide, not a marketing post.